numpy/core/arrayprint.py

```diff
@@ -715,16 +711,9 @@ def array2string(a, max_line_width=None, precision=None,
     options.update(overrides)
 
     if options['legacy'] <= 113:
-        if style is np._NoValue:
-            style = repr
```

Comment:
Staring at the code for other reasons, we still did support it here in "legacy" mode.  I suppose that could probably have been deprecated here also?  Or can we deprecate the legacy `"1.13"` mode itself eventually?

Reply:
ahh, that changes everything, so basically it has to be kept in the signature to support the legacy mode.